internal/db/schema/migrations/oss/postgres/48/03_create_kms_key_destructions.up.sql

```diff
@@ -0,0 +1,40 @@
+begin;
+
+  create table kms_key_destruction_status_enm (
+    name text primary key
+      constraint only_predefined_values_allowed
+      check(name in ('pending', 'running', 'completed', 'failed'))
+  );
+  comment on table kms_key_destruction_status_enm is
+    'Table holding valid statuses a key destruction can take';
+
+  insert into kms_key_destruction_status_enm (name)
+  values
+    ('pending'),
+    ('running'),
+    ('completed'),
+    ('failed');
+
+  create trigger 
+    immutable_columns
+  before
+  update on kms_key_destruction_status_enm
+    for each row execute procedure immutable_columns('name');
```

Comment:
```suggestion
   create trigger immutable_columns before update on kms_key_destruction_status_enm
    for each row execute procedure immutable_columns('name');
```
Please reformat to follow our [sql style guide](https://github.com/hashicorp/boundary/blob/main/sql-style-guide.md#triggers).